src/env.js

```diff
@@ -4,6 +4,7 @@ export const ENV = {
     LOCAL:      ('local' : 'local'),
     STAGE:      ('stage' : 'stage'),
     SANDBOX:    ('sandbox' : 'sandbox'),
+    SANDBOX_V2: ('sandbox-v2' : 'sandbox-v2'),
```

Comment:
@gabrielo91 This is really small but I think we want the value to be `sandbox_v2`.  We use dashes for the attributes and think from a styling it may be good to use underscore for values to differentiate better.  @gregjopa Thoughts?

Reply:
The dreaded space 😄. I do prefer `sandbox_v2` for the value.

Reply:
good cached, let me fix it